packages/css-data/src/property-parsers/text-shadow-property-extractor.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  KeywordValue,
+  RgbValue,
+  TupleValue,
+  TupleValueItem,
+  UnitValue,
+} from "@webstudio-is/css-engine";
+
+/*
+  extractBoxShadowProperties accepts a TupleValue
+  and just tokenizes the valuess of box-shadow items.
+  https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow#syntax
+
+  To validate the text-shadow string value, should be used
+  in combination with parseShadow function.
+
+  Eg:
+  extractBoxShadowProperties(parseShadow('boxShadow', "5em 5em 5em 5em #ffff"))
+*/
+
+export type ExtractedTextShadowProperties = {
+  color?: RgbValue | KeywordValue;
+  offsetX: UnitValue;
+  offsetY: UnitValue;
+  blur: UnitValue;
+};
+
+export const extractTextShadowProperties = (
```

Comment:
is it different from box shadow extractor?

Reply:
Yes, `text-shadow` don't have `blur` and `spread`. And really didn't want to mix the extractors just in case if we need to modify them if needed too.

Reply:
@TrySound merged both the extractors now. Let's see if we really need to be separated down the line then.